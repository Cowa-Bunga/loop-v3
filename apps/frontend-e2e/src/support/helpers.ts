export const testIdSelector = (testId: string) => `[data-testid="${testId}"]`

export const COOKIE_NAME = 'cowabunga-user-cookie'
