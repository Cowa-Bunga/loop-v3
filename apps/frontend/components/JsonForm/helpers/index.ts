const makeInputId = (id: string, suffix?: string) => {
  const parts = id.split('/')

  if (suffix) {
    return `${parts[parts.length - 1]}-${suffix}`
  }

  return parts[parts.length - 1]
}

export { makeInputId }
