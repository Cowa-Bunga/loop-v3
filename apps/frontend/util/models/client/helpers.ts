import { defaultsDeep } from 'lodash'
import { MODEL_CLIENT_EMPTY } from './model'

export const modelClientList = (
  data: { [s: string]: unknown } | ArrayLike<unknown>
): IClient[] => {
  return Object.entries(data)
    .map(modelClientData)
    .map((el: never) => el['1'])
    .sort((a: IClient, b: IClient) => a.name.localeCompare(b.name))
}

export const modelClientData = (data: unknown): IClient => {
  return defaultsDeep(data, MODEL_CLIENT_EMPTY)
}
