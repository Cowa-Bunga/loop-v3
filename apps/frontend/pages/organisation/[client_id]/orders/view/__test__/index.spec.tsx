import Orders from '../index.page'
import { renderWithProviders } from '@test/helpers'

describe('Orders', function () {
  it('should render successfully', () => {
    const { baseElement } = renderWithProviders(<Orders />)

    expect(baseElement).toBeTruthy()
    expect(baseElement).toMatchSnapshot()
  })
})
