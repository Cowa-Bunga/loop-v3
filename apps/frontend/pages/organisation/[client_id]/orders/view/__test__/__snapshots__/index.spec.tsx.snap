// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Orders should render successfully 1`] = `
<body>
  <div>
    <div
      class="MuiBox-root css-i9gxme"
    >
      <header
        class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionFixed mui-fixed css-o0v5hs-MuiPaper-root-MuiAppBar-root"
      >
        <div
          class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-hyum1k-MuiToolbar-root"
        >
          <div
            class="MuiTypography-root MuiTypography-h6 css-1qvppxn-MuiTypography-root"
          >
            <img
              alt="logo"
              src="https://www.loop.co.za/wp-content/uploads/2021/12/Logo.svg"
              style="margin-top: 5px; height: 20px;"
            />
          </div>
          <button
            class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit MuiIconButton-sizeMedium css-zylse7-MuiButtonBase-root-MuiIconButton-root"
            tabindex="0"
            type="button"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
              data-testid="LockIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M18 8h-1V6c0-2.76-2.24-5-5-5S7 3.24 7 6v2H6c-1.1 0-2 .9-2 2v10c0 1.1.9 2 2 2h12c1.1 0 2-.9 2-2V10c0-1.1-.9-2-2-2zm-6 9c-1.1 0-2-.9-2-2s.9-2 2-2 2 .9 2 2-.9 2-2 2zm3.1-9H8.9V6c0-1.71 1.39-3.1 3.1-3.1 1.71 0 3.1 1.39 3.1 3.1v2z"
              />
            </svg>
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
        </div>
      </header>
    </div>
    <div
      class="MuiBox-root css-1t019bo"
    >
      <div
        class="MuiBox-root css-1mb7ed4"
      >
        <div
          class="MuiBox-root css-0"
        >
          <h6
            class="MuiTypography-root MuiTypography-h6 css-2ulfj5-MuiTypography-root"
          >
            Orders
          </h6>
        </div>
        <br />
        <hr
          class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
        />
        <br />
        <div
          class="MuiTabs-root css-1ujnqem-MuiTabs-root"
        >
          <div
            class="MuiTabs-scroller MuiTabs-fixed css-jpln7h-MuiTabs-scroller"
            style="overflow: hidden; margin-bottom: 0px;"
          >
            <div
              class="MuiTabs-flexContainer css-heg063-MuiTabs-flexContainer"
              role="tablist"
            >
              <button
                aria-selected="true"
                class="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary Mui-selected css-1h9z7r5-MuiButtonBase-root-MuiTab-root"
                role="tab"
                tabindex="0"
                type="button"
              >
                Incomplete
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
              <button
                aria-selected="false"
                class="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary css-1h9z7r5-MuiButtonBase-root-MuiTab-root"
                role="tab"
                tabindex="-1"
                type="button"
              >
                Completed
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
              <button
                aria-selected="false"
                class="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary css-1h9z7r5-MuiButtonBase-root-MuiTab-root"
                role="tab"
                tabindex="-1"
                type="button"
              >
                Abandoned
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
              <button
                aria-selected="false"
                class="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary css-1h9z7r5-MuiButtonBase-root-MuiTab-root"
                role="tab"
                tabindex="-1"
                type="button"
              >
                Unassignable
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
            </div>
            <span
              class="MuiTabs-indicator css-1aquho2-MuiTabs-indicator"
              style="left: 0px; width: 0px;"
            />
          </div>
        </div>
        <div
          aria-labelledby="tab-0"
          class="MuiBox-root css-0"
          id="tabpanel-0"
          role="tabpanel"
        >
          <div
            class="MuiBox-root css-0"
          >
            <div
              aria-colcount="9"
              aria-multiselectable="false"
              aria-rowcount="4"
              class="MuiDataGrid-root MuiDataGrid-autoHeight MuiDataGrid-root--densityComfortable MuiDataGrid-withBorderColor css-1usso8h-MuiDataGrid-root"
              role="grid"
            >
              <div />
              <div
                class="MuiDataGrid-main css-204u17-MuiDataGrid-main"
              >
                <div
                  class="MuiDataGrid-columnHeaders MuiDataGrid-withBorderColor css-1iyq7zh-MuiDataGrid-columnHeaders"
                  style="min-height: 72px; max-height: 72px; line-height: 72px;"
                >
                  <div
                    class="MuiDataGrid-columnHeadersInner MuiDataGrid-columnHeadersInner--scrollable css-gl260s-MuiDataGrid-columnHeadersInner"
                    role="rowgroup"
                    style="transform: translate3d(0px, 0px, 0px);"
                  >
                    <div
                      aria-rowindex="1"
                      class="css-yrdy0g-MuiDataGrid-columnHeaderRow"
                      role="row"
                    >
                      <div
                        aria-colindex="1"
                        aria-label="ORDER NUMBER"
                        aria-sort="none"
                        class="MuiDataGrid-columnHeader MuiDataGrid-columnHeader--sortable MuiDataGrid-withBorderColor"
                        data-field="col1"
                        role="columnheader"
                        style="height: 72px; width: 180px; min-width: 180px; max-width: 180px;"
                        tabindex="0"
                      >
                        <div
                          class="MuiDataGrid-columnHeaderDraggableContainer"
                          draggable="false"
                        >
                          <div
                            class="MuiDataGrid-columnHeaderTitleContainer"
                          >
                            <div
                              class="MuiDataGrid-columnHeaderTitleContainerContent"
                            >
                              <div
                                aria-label=""
                                class="MuiDataGrid-columnHeaderTitle css-t89xny-MuiDataGrid-columnHeaderTitle"
                                data-mui-internal-clone-element="true"
                              >
                                ORDER NUMBER
                              </div>
                            </div>
                            <div
                              class="MuiDataGrid-iconButtonContainer css-ltf0zy-MuiDataGrid-iconButtonContainer"
                            >
                              <button
                                aria-label="Sort"
                                class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeSmall css-1pe4mpk-MuiButtonBase-root-MuiIconButton-root"
                                tabindex="-1"
                                title="Sort"
                                type="button"
                              >
                                <svg
                                  aria-hidden="true"
                                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall MuiDataGrid-sortIcon css-ptiqhd-MuiSvgIcon-root"
                                  data-testid="ArrowUpwardIcon"
                                  focusable="false"
                                  viewBox="0 0 24 24"
                                >
                                  <path
                                    d="M4 12l1.41 1.41L11 7.83V20h2V7.83l5.58 5.59L20 12l-8-8-8 8z"
                                  />
                                </svg>
                                <span
                                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                                />
                              </button>
                            </div>
                          </div>
                          <div
                            class="MuiDataGrid-menuIcon"
                          >
                            <button
                              aria-controls=":r0:"
                              aria-haspopup="true"
                              aria-label="Menu"
                              class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeSmall MuiDataGrid-menuIconButton css-1pe4mpk-MuiButtonBase-root-MuiIconButton-root"
                              data-mui-internal-clone-element="true"
                              id=":r1:"
                              tabindex="-1"
                              type="button"
                            >
                              <svg
                                aria-hidden="true"
                                class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall css-ptiqhd-MuiSvgIcon-root"
                                data-testid="TripleDotsVerticalIcon"
                                focusable="false"
                                viewBox="0 0 24 24"
                              >
                                <path
                                  d="M12 8c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2zm0 2c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm0 6c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z"
                                />
                              </svg>
                              <span
                                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                              />
                            </button>
                          </div>
                        </div>
                        <div
                          class="MuiDataGrid-columnSeparator MuiDataGrid-columnSeparator--sideRight"
                          style="min-height: 72px; opacity: 1;"
                        >
                          <svg
                            aria-hidden="true"
                            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiDataGrid-iconSeparator css-i4bv87-MuiSvgIcon-root"
                            data-testid="SeparatorIcon"
                            focusable="false"
                            viewBox="0 0 24 24"
                          >
                            <path
                              d="M11 19V5h2v14z"
                            />
                          </svg>
                        </div>
                      </div>
                      <div
                        aria-colindex="2"
                        aria-label="CUSTOMER NAME"
                        aria-sort="none"
                        class="MuiDataGrid-columnHeader MuiDataGrid-columnHeader--sortable MuiDataGrid-withBorderColor"
                        data-field="col2"
                        role="columnheader"
                        style="height: 72px; width: 180px; min-width: 180px; max-width: 180px;"
                        tabindex="-1"
                      >
                        <div
                          class="MuiDataGrid-columnHeaderDraggableContainer"
                          draggable="false"
                        >
                          <div
                            class="MuiDataGrid-columnHeaderTitleContainer"
                          >
                            <div
                              class="MuiDataGrid-columnHeaderTitleContainerContent"
                            >
                              <div
                                aria-label=""
                                class="MuiDataGrid-columnHeaderTitle css-t89xny-MuiDataGrid-columnHeaderTitle"
                                data-mui-internal-clone-element="true"
                              >
                                CUSTOMER NAME
                              </div>
                            </div>
                            <div
                              class="MuiDataGrid-iconButtonContainer css-ltf0zy-MuiDataGrid-iconButtonContainer"
                            >
                              <button
                                aria-label="Sort"
                                class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeSmall css-1pe4mpk-MuiButtonBase-root-MuiIconButton-root"
                                tabindex="-1"
                                title="Sort"
                                type="button"
                              >
                                <svg
                                  aria-hidden="true"
                                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall MuiDataGrid-sortIcon css-ptiqhd-MuiSvgIcon-root"
                                  data-testid="ArrowUpwardIcon"
                                  focusable="false"
                                  viewBox="0 0 24 24"
                                >
                                  <path
                                    d="M4 12l1.41 1.41L11 7.83V20h2V7.83l5.58 5.59L20 12l-8-8-8 8z"
                                  />
                                </svg>
                                <span
                                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                                />
                              </button>
                            </div>
                          </div>
                          <div
                            class="MuiDataGrid-menuIcon"
                          >
                            <button
                              aria-controls=":r4:"
                              aria-haspopup="true"
                              aria-label="Menu"
                              class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeSmall MuiDataGrid-menuIconButton css-1pe4mpk-MuiButtonBase-root-MuiIconButton-root"
                              data-mui-internal-clone-element="true"
                              id=":r5:"
                              tabindex="-1"
                              type="button"
                            >
                              <svg
                                aria-hidden="true"
                                class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall css-ptiqhd-MuiSvgIcon-root"
                                data-testid="TripleDotsVerticalIcon"
                                focusable="false"
                                viewBox="0 0 24 24"
                              >
                                <path
                                  d="M12 8c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2zm0 2c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm0 6c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z"
                                />
                              </svg>
                              <span
                                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                              />
                            </button>
                          </div>
                        </div>
                        <div
                          class="MuiDataGrid-columnSeparator MuiDataGrid-columnSeparator--sideRight"
                          style="min-height: 72px; opacity: 1;"
                        >
                          <svg
                            aria-hidden="true"
                            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiDataGrid-iconSeparator css-i4bv87-MuiSvgIcon-root"
                            data-testid="SeparatorIcon"
                            focusable="false"
                            viewBox="0 0 24 24"
                          >
                            <path
                              d="M11 19V5h2v14z"
                            />
                          </svg>
                        </div>
                      </div>
                      <div
                        aria-colindex="3"
                        aria-label="TIME PLACED"
                        aria-sort="none"
                        class="MuiDataGrid-columnHeader MuiDataGrid-columnHeader--sortable MuiDataGrid-withBorderColor"
                        data-field="col3"
                        role="columnheader"
                        style="height: 72px; width: 180px; min-width: 180px; max-width: 180px;"
                        tabindex="-1"
                      >
                        <div
                          class="MuiDataGrid-columnHeaderDraggableContainer"
                          draggable="false"
                        >
                          <div
                            class="MuiDataGrid-columnHeaderTitleContainer"
                          >
                            <div
                              class="MuiDataGrid-columnHeaderTitleContainerContent"
                            >
                              <div
                                aria-label=""
                                class="MuiDataGrid-columnHeaderTitle css-t89xny-MuiDataGrid-columnHeaderTitle"
                                data-mui-internal-clone-element="true"
                              >
                                TIME PLACED
                              </div>
                            </div>
                            <div
                              class="MuiDataGrid-iconButtonContainer css-ltf0zy-MuiDataGrid-iconButtonContainer"
                            >
                              <button
                                aria-label="Sort"
                                class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeSmall css-1pe4mpk-MuiButtonBase-root-MuiIconButton-root"
                                tabindex="-1"
                                title="Sort"
                                type="button"
                              >
                                <svg
                                  aria-hidden="true"
                                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall MuiDataGrid-sortIcon css-ptiqhd-MuiSvgIcon-root"
                                  data-testid="ArrowUpwardIcon"
                                  focusable="false"
                                  viewBox="0 0 24 24"
                                >
                                  <path
                                    d="M4 12l1.41 1.41L11 7.83V20h2V7.83l5.58 5.59L20 12l-8-8-8 8z"
                                  />
                                </svg>
                                <span
                                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                                />
                              </button>
                            </div>
                          </div>
                          <div
                            class="MuiDataGrid-menuIcon"
                          >
                            <button
                              aria-controls=":r8:"
                              aria-haspopup="true"
                              aria-label="Menu"
                              class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeSmall MuiDataGrid-menuIconButton css-1pe4mpk-MuiButtonBase-root-MuiIconButton-root"
                              data-mui-internal-clone-element="true"
                              id=":r9:"
                              tabindex="-1"
                              type="button"
                            >
                              <svg
                                aria-hidden="true"
                                class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall css-ptiqhd-MuiSvgIcon-root"
                                data-testid="TripleDotsVerticalIcon"
                                focusable="false"
                                viewBox="0 0 24 24"
                              >
                                <path
                                  d="M12 8c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2zm0 2c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm0 6c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z"
                                />
                              </svg>
                              <span
                                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                              />
                            </button>
                          </div>
                        </div>
                        <div
                          class="MuiDataGrid-columnSeparator MuiDataGrid-columnSeparator--sideRight"
                          style="min-height: 72px; opacity: 1;"
                        >
                          <svg
                            aria-hidden="true"
                            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiDataGrid-iconSeparator css-i4bv87-MuiSvgIcon-root"
                            data-testid="SeparatorIcon"
                            focusable="false"
                            viewBox="0 0 24 24"
                          >
                            <path
                              d="M11 19V5h2v14z"
                            />
                          </svg>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
                <div
                  style="flex: 0 0px; overflow: visible;"
                >
                  <div
                    class="MuiDataGrid-virtualScroller css-axafay-MuiDataGrid-virtualScroller"
                    style="overflow-x: hidden; overflow-y: hidden;"
                  >
                    <div
                      class="MuiDataGrid-virtualScrollerContent css-1kwdphh-MuiDataGrid-virtualScrollerContent"
                      style="width: auto; height: 201px; min-height: auto;"
                    >
                      <div
                        class="MuiDataGrid-virtualScrollerRenderZone css-s1v7zr-MuiDataGrid-virtualScrollerRenderZone"
                        style="transform: translate3d(0px, 0px, 0px);"
                      >
                        <div
                          aria-rowindex="2"
                          aria-selected="false"
                          class="MuiDataGrid-row"
                          data-id="1"
                          data-rowindex="0"
                          role="row"
                          style="max-height: 67px; min-height: 67px;"
                        >
                          <div
                            aria-colindex="1"
                            aria-colspan="1"
                            class="MuiDataGrid-cell MuiDataGrid-cell--textLeft MuiDataGrid-withBorderColor"
                            data-colindex="0"
                            data-field="col1"
                            role="cell"
                            style="min-width: 180px; max-width: 180px; min-height: 67px; max-height: 67px;"
                            tabindex="-1"
                          >
                            <div
                              class="MuiDataGrid-cellContent"
                              title="ORDER001111"
                            >
                              ORDER001111
                            </div>
                          </div>
                          <div
                            aria-colindex="2"
                            aria-colspan="1"
                            class="MuiDataGrid-cell MuiDataGrid-cell--textLeft MuiDataGrid-withBorderColor"
                            data-colindex="1"
                            data-field="col2"
                            role="cell"
                            style="min-width: 180px; max-width: 180px; min-height: 67px; max-height: 67px;"
                            tabindex="-1"
                          >
                            <div
                              class="MuiDataGrid-cellContent"
                              title="John"
                            >
                              John
                            </div>
                          </div>
                          <div
                            aria-colindex="3"
                            aria-colspan="1"
                            class="MuiDataGrid-cell MuiDataGrid-cell--textLeft MuiDataGrid-withBorderColor"
                            data-colindex="2"
                            data-field="col3"
                            role="cell"
                            style="min-width: 180px; max-width: 180px; min-height: 67px; max-height: 67px;"
                            tabindex="-1"
                          >
                            <div
                              class="MuiDataGrid-cellContent"
                              title="23/3/2023 14:21:33"
                            >
                              23/3/2023 14:21:33
                            </div>
                          </div>
                        </div>
                        <div
                          aria-rowindex="3"
                          aria-selected="false"
                          class="MuiDataGrid-row"
                          data-id="2"
                          data-rowindex="1"
                          role="row"
                          style="max-height: 67px; min-height: 67px;"
                        >
                          <div
                            aria-colindex="1"
                            aria-colspan="1"
                            class="MuiDataGrid-cell MuiDataGrid-cell--textLeft MuiDataGrid-withBorderColor"
                            data-colindex="0"
                            data-field="col1"
                            role="cell"
                            style="min-width: 180px; max-width: 180px; min-height: 67px; max-height: 67px;"
                            tabindex="-1"
                          >
                            <div
                              class="MuiDataGrid-cellContent"
                              title="ORDER001211"
                            >
                              ORDER001211
                            </div>
                          </div>
                          <div
                            aria-colindex="2"
                            aria-colspan="1"
                            class="MuiDataGrid-cell MuiDataGrid-cell--textLeft MuiDataGrid-withBorderColor"
                            data-colindex="1"
                            data-field="col2"
                            role="cell"
                            style="min-width: 180px; max-width: 180px; min-height: 67px; max-height: 67px;"
                            tabindex="-1"
                          >
                            <div
                              class="MuiDataGrid-cellContent"
                              title="Sally"
                            >
                              Sally
                            </div>
                          </div>
                          <div
                            aria-colindex="3"
                            aria-colspan="1"
                            class="MuiDataGrid-cell MuiDataGrid-cell--textLeft MuiDataGrid-withBorderColor"
                            data-colindex="2"
                            data-field="col3"
                            role="cell"
                            style="min-width: 180px; max-width: 180px; min-height: 67px; max-height: 67px;"
                            tabindex="-1"
                          >
                            <div
                              class="MuiDataGrid-cellContent"
                              title="23/3/2023 14:21:33"
                            >
                              23/3/2023 14:21:33
                            </div>
                          </div>
                        </div>
                        <div
                          aria-rowindex="4"
                          aria-selected="false"
                          class="MuiDataGrid-row MuiDataGrid-row--lastVisible"
                          data-id="3"
                          data-rowindex="2"
                          role="row"
                          style="max-height: 67px; min-height: 67px;"
                        >
                          <div
                            aria-colindex="1"
                            aria-colspan="1"
                            class="MuiDataGrid-cell MuiDataGrid-cell--textLeft MuiDataGrid-withBorderColor"
                            data-colindex="0"
                            data-field="col1"
                            role="cell"
                            style="min-width: 180px; max-width: 180px; min-height: 67px; max-height: 67px;"
                            tabindex="-1"
                          >
                            <div
                              class="MuiDataGrid-cellContent"
                              title="ORDER001113"
                            >
                              ORDER001113
                            </div>
                          </div>
                          <div
                            aria-colindex="2"
                            aria-colspan="1"
                            class="MuiDataGrid-cell MuiDataGrid-cell--textLeft MuiDataGrid-withBorderColor"
                            data-colindex="1"
                            data-field="col2"
                            role="cell"
                            style="min-width: 180px; max-width: 180px; min-height: 67px; max-height: 67px;"
                            tabindex="-1"
                          >
                            <div
                              class="MuiDataGrid-cellContent"
                              title="Robert"
                            >
                              Robert
                            </div>
                          </div>
                          <div
                            aria-colindex="3"
                            aria-colspan="1"
                            class="MuiDataGrid-cell MuiDataGrid-cell--textLeft MuiDataGrid-withBorderColor"
                            data-colindex="2"
                            data-field="col3"
                            role="cell"
                            style="min-width: 180px; max-width: 180px; min-height: 67px; max-height: 67px;"
                            tabindex="-1"
                          >
                            <div
                              class="MuiDataGrid-cellContent"
                              title="23/3/2023 14:21:33"
                            >
                              23/3/2023 14:21:33
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
                <div
                  class="Mui-resizeTriggers"
                >
                  <div
                    class="expand-trigger"
                  >
                    <div
                      style="width: 1px; height: 1px;"
                    />
                  </div>
                  <div
                    class="contract-trigger"
                  />
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          aria-labelledby="tab-1"
          class="MuiBox-root css-0"
          hidden=""
          id="tabpanel-1"
          role="tabpanel"
        />
        <div
          aria-labelledby="tab-2"
          class="MuiBox-root css-0"
          hidden=""
          id="tabpanel-2"
          role="tabpanel"
        />
      </div>
    </div>
  </div>
</body>
`;
